biojava-alignment/src/main/java/org/biojava/nbio/alignment/SimpleGapPenalty.java

```diff
@@ -30,9 +32,13 @@
  *
  * @author Mark Chapman
  */
-public class SimpleGapPenalty implements GapPenalty {
+public class SimpleGapPenalty implements GapPenalty, Serializable {
```

Comment:
wish this pull request hadn't been merged so quickly, making non-final public classes Serializable is somewhat dangerous, see e.g. the chapter on serialization in Effective Java.